conans/tools.py

```diff
@@ -10,6 +10,7 @@
 import requests
 from conans.client.output import ConanOutput
 import platform
+if platform.system() == "Linux": import distro
```

Comment:
Remove this line and move the import to line 255 (inside if self.is_linux block)